articat/tests/catalog_local_test.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from articat.catalog_local import CatalogLocal
+from articat.tests.utils import TestFSArtifact
+from articat.catalog import MissingArtifactException  # Ensure MissingArtifactException is imported
+
+def test_catalog_local_missing_artifact_exception():
+    """
+    Test that `MissingArtifactException` is raised when an artifact cannot be found in `CatalogLocal`.
+    """
+    catalog = CatalogLocal()
+    with pytest.raises(MissingArtifactException) as exc_info:
```

Comment:
It's more idiomatic to use `match` argument on `pytest.raises` to validate a message, just like in the other tests